components/docker-auth/cmd/authn/authentication.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/golang/glog"
+
+	"github.com/cellery-io/cellery-hub/components/docker-auth/pkg"
+)
+
+var (
+	idpKey        []byte
+	dockerAuthKey []byte
+	err           error
+)
+
+const (
+	issuerClaim = "iss"
+	subjectClaim = "sub"
+	expirationClaim = "exp"
+	idpCertEnvVar = "IDP_CERT"
+	authTokenIssuer = "REGISTRY_AUTH_TOKEN_ISSUER"
+	dockerAuthCertEnvVar = "REGISTRY_AUTH_TOKEN_ROOTCERTBUNDLE"
+)
+
+func errLog(err error) {
+	if err != nil {
+		log.Fatal("Error:", err.Error())
+	}
+}
+
+func init() {
+	flag.Usage = usage
```

Comment:
You can directly pass the function without defining it separately since it is not used by anything else.